addons/mail/static/src/components/attachment_delete_confirm/attachment_delete_confirm.js

```diff
@@ -2,16 +2,13 @@
 
 import { registerMessagingComponent } from '@mail/utils/messaging_component';
 import { useComponentToModel } from '@mail/component_hooks/use_component_to_model/use_component_to_model';
+import { escape } from '@web/core/utils/strings';
 
 const { Component } = owl;
 
 export class AttachmentDeleteConfirm extends Component {
-
-    /**
-     * @override
-     */
     setup() {
-        super.setup();
```

Comment:
Also not these

Reply:
This call is useless and doesn't follow the not-yet-official guideline: "do not call super in components extending Component directly"